db/schema.rb

```diff
@@ -87,32 +87,30 @@
   end
 
   create_table "contacts", force: :cascade do |t|
-    t.integer  "user_id"
-    t.string   "firstname"
-    t.string   "lastname"
-    t.date     "date_of_birth"
-    t.string   "place_of_birth"
     t.string   "phone1"
     t.string   "phone2"
     t.string   "club_email"
-    t.datetime "created_at",     null: false
-    t.datetime "updated_at",     null: false
-    t.index ["user_id"], name: "index_contacts_on_user_id", using: :btree
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer  "member_id"
+    t.index ["member_id"], name: "index_contacts_on_member_id", using: :btree
   end
 
   create_table "events", force: :cascade do |t|
+    t.datetime "datetime",                      null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "description",   limit: 255
-    t.string   "address",       limit: 255,                 null: false
-    t.string   "name",                                      null: false
+    t.string   "description"
+    t.string   "address",                       null: false
+    t.string   "home_team"
+    t.string   "away_team"
```

Comment:
After research, I think the `schema.rb` in master branch is wrong and the fields just missing